docs/admin/federation-apiserver.md

```diff
@@ -1,8 +1,3 @@
 This file is autogenerated, but we've stopped checking such files into the
 repository to reduce the need for rebases. Please run hack/generate-docs.sh to
 populate this file.
-
```

Comment:
As a matter of proceeding to make the change easier to read, should I pushed two commits instead of one?  I made my change, committed, then ran hack/update-all.sh, then committed again, then rebased the two into one, then pushed.